openhcl/underhill_attestation/src/key_protector.rs

```diff
@@ -117,7 +119,7 @@ impl KeyProtectorExt for KeyProtector {
             }
 
             let rsa_unwrapped_key = if let Some(wrapped_des_key) = wrapped_des_key {
-                tracing::info!("wrapped key is present");
+                tracing::info!(CVM_ALLOWED, "wrapped key is present");
```

Comment:
Could you share the source of this tracing function?

Reply:
This is the standard function from rust https://docs.rs/tracing/latest/tracing/